### PR TITLE
Revert "Update readme file repo"

### PR DESCRIPTION
**Pull request by jhonnelrem**
Reverts jhonnelrem/Test-Repo-Git#4

### CODE REVIEW

**Comment by jhonerApp** [comment]
Update readme

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-Update readmd to fix minor issue in my local repo.
```
Comment (jhonerApp) [inline README.md:1]: Mark as this fix

**Review by jhonerApp — COMMENTED** [review]
Commit my fix

File: README.md (hunk 1 above)
Comment (jhonnelrem) [inline README.md:1]: dfg

**Review by jhonnelrem — COMMENTED** [review]
Finised

**Pull request merged**